release-notes/10.0/preview/preview2/api-diff/Microsoft.WindowsDesktop.App/10.0-preview2_System.Windows.Forms.Design.md

```diff
@@ -0,0 +1,50 @@
+# System.Windows.Forms.Design
+
+```diff
+  namespace System.ComponentModel.Design
+  {
+      public class ArrayEditor : CollectionEditor
+      {
+-         protected override System.Object[] GetItems(System.Object editValue) { throw null; }
+-         protected override System.Object SetItems(System.Object editValue, System.Object[] value) { throw null; }
++         protected override System.Object[] GetItems(System.Object? editValue) { throw null; }
++         protected override System.Object? SetItems(System.Object? editValue, System.Object[]? value) { throw null; }
+      }
+      public class InheritanceService
+      {
+-         protected virtual System.Boolean IgnoreInheritedMember(System.Reflection.MemberInfo member, ComponentModel.IComponent? component) { throw null; }
++         protected virtual System.Boolean IgnoreInheritedMember(Reflection.MemberInfo member, ComponentModel.IComponent? component) { throw null; }
+      }
+  }
+  namespace System.ComponentModel.Design.Serialization
+  {
+      public class CollectionCodeDomSerializer : CodeDomSerializer
+      {
+-         protected System.Boolean MethodSupportsSerialization(System.Reflection.MethodInfo method) { throw null; }
++         protected System.Boolean MethodSupportsSerialization(Reflection.MethodInfo method) { throw null; }
+      }
+  }
+  namespace System.Drawing.Design
+  {
+      public class ToolboxItem
+      {
+-         public System.Reflection.AssemblyName? AssemblyName { get { throw null; } set { } }
++         public Reflection.AssemblyName? AssemblyName { get { throw null; } set { } }
+-         public System.Reflection.AssemblyName[]? DependentAssemblies { get { throw null; } set { } }
++         public Reflection.AssemblyName[]? DependentAssemblies { get { throw null; } set { } }
+-         protected virtual System.Type? GetType(ComponentModel.Design.IDesignerHost? host, System.Reflection.AssemblyName? assemblyName, System.String typeName, System.Boolean? reference) { throw null; }
++         protected virtual System.Type? GetType(ComponentModel.Design.IDesignerHost? host, Reflection.AssemblyName? assemblyName, System.String typeName, System.Boolean? reference) { throw null; }
+      }
+  }
+  namespace System.Windows.Forms.Design
```

Comment:
I think we had added some classes to this namespace, I need to double check that.

Reply:
Sounds good. Let me know.

Reply:
no, we shipped more classes in preview 3